server/src/ga-activation.js

```diff
@@ -21,7 +21,7 @@ function _dntEnabled(dnt, ua) {
     // List of Windows versions known to not implement DNT according to the standard.
     var anomalousWinVersions = ['Windows NT 6.1', 'Windows NT 6.2', 'Windows NT 6.3'];
 
-    var fxMatch = ua.match(/Firefox\\/(\\d+)/);
+    var fxMatch = ua.match(/Firefox\\/(\\d{1,4000})/);
```

Comment:
This is a Firefox version, so I think we can do something more limited like `\\d{1,10}`